### PR TITLE
feature: fix Path level Parameters

### PR DESCRIPTION
**Pull request by klokane**
Related to changes in fury-adapter-swagger#98

### CODE REVIEW

File: fixtures/features/api-elements/param-with-enum-default.json [hunk 1]
```diff
@@ -86,7 +86,9 @@
                           "meta": {},
                           "attributes": {
                             "default": [
-                              "A"
```
Comment (pksunkara) [inline fixtures/features/api-elements/param-with-enum-default.json]: Was this a bug in swagger adapter? Enum's default is an array in APIB refract, right?

Comment (klokane) [reply fixtures/features/api-elements/param-with-enum-default.json]: Not sure, what happen there. It is "regenerated test"
Will investigate why it happen.

**Comment by pksunkara** [comment]
@klokane Please add a release commit too and I can merge this.

File: fixtures/features/api-elements/form-data.json [hunk 2]
```diff
@@ -96,7 +96,7 @@
                           "attributes": {
                             "contentType": "application/x-www-form-urlencoded"
                           },
-                          "content": "id=1&bar=chocolate-bar"
+                          "content": "id=1&bar=chocolate-bar&foo=esse%20do%20Excepteur%20sin"
```
Comment (pksunkara) [inline fixtures/features/api-elements/form-data.json]: We can't use the auto generation. We need an enum value to fix this.

**Review by pksunkara — CHANGES_REQUESTED** [review]

**Review by pksunkara — APPROVED** [review]

**Pull request merged**